src/main/java/graphql/execution/instrumentation/ChainedInstrumentation.java

```diff
@@ -126,6 +131,11 @@ public ExecutionStrategyInstrumentationContext beginExecutionStrategy(Instrument
         }));
     }
 
+    @Override
+    public InstrumentationContext<ExecutionResult> beginField(InstrumentationFieldParameters parameters, InstrumentationState state) {
+        return new ChainedInstrumentationContext<>(map(instrumentations, it -> it.beginField(parameters, state)));
+    }
+
```

Comment:
Note I still need to create a wrapper ChainedInstrumentationContext object per call

The alternative is

```
        ImmutableList<InstrumentationContext<Object>> contexts = map(instrumentations, it -> it.beginFieldFetch(parameters, getState(it, state)));
        return new InstrumentationContext<Object>() {
            @Override
            public void onDispatched(CompletableFuture<Object> result) {
                contexts.forEach(ctx -> ctx.onDispatched(result));
            }

            @Override
            public void onCompleted(Object result, Throwable t) {
                contexts.forEach(ctx -> ctx.onCompleted(result,t));
            }
        };
```

and this is an object allocation anyway - since we MUST call back when the result is dispatched and most importantly when its finished